CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## 1.1.0 - 2017-12-15
```

Comment:
I think we were trying to keep with the format at [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).